### ISSUE TITLE
Handle Micka error when getting info about metadata record

### ISSUE THREAD

**Issue by jirik** [jirik/layman#43]
```
[2020-02-14 09:23:14 +0000] [13] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/usr/local/lib/python3.6/dist-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/usr/local/lib/python3.6/dist-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/usr/local/lib/python3.6/dist-packages/gunicorn/workers/sync.py", line 176, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 2463, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 2449, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1866, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.6/dist-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.6/dist-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1947, in full_dispatch_request
    rv = self.preprocess_request()
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 2241, in preprocess_request
    rv = func()
  File "/code/src/layman/authn/util.py", line 22, in decorated_function
    return f(*args, **kwargs)
  File "/code/src/layman/authz/util.py", line 17, in decorated_function
    return f(*args, **kwargs)
  File "/code/src/layman/util.py", line 53, in decorated_function
    result = f(*args, **kwargs)
  File "/code/src/layman/layer/util.py", line 31, in decorated_function
    result = f(*args, **kwargs)
  File "/code/src/layman/layer/util.py", line 41, in decorated_function
    info = get_complete_layer_info(username, layername)
  File "/code/src/layman/layer/util.py", line 123, in get_complete_layer_info
    partial_info = get_layer_info(username, layername)
  File "/code/src/layman/layer/util.py", line 88, in get_layer_info
    partial_infos = call_modules_fn(sources, 'get_layer_info', [username, layername])
  File "/code/src/layman/util.py", line 171, in call_modules_fn
    k:v for k, v in kwargs.items()
  File "/code/src/layman/layer/micka/csw.py", line 28, in get_layer_info
    csw = util.create_csw()
  File "/code/src/layman/layer/micka/util.py", line 84, in create_csw
    csw = CatalogueServiceWeb(settings.CSW_URL, **opts) if settings.CSW_URL is not None else None
  File "/usr/local/lib/python3.6/dist-packages/owslib/csw.py", line 85, in __init__
    self._invoke()
  File "/usr/local/lib/python3.6/dist-packages/owslib/csw.py", line 671, in _invoke
    self.request, None, 'Get', timeout=self.timeout, auth=self.auth
  File "/usr/local/lib/python3.6/dist-packages/owslib/util.py", line 211, in openURL
    req.raise_for_status()
  File "/usr/local/lib/python3.6/dist-packages/requests/models.py", line 940, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: https://dev.bnhelp.cz/projects/micka6/csw?service=CSW&version=2.0.2&request=GetCapabilities

```

**Comment by jirik** [comment]
1. Create branches v1.1.x and v1.2.x
1. Catch HTTPError at
    - https://github.com/jirik/layman/blob/master/src/layman/layer/micka/csw.py#L28
    - https://github.com/jirik/layman/blob/master/src/layman/layer/micka/csw.py#L32
      and return empty object

**Comment by jirik** [comment]
Closed by 5429d2b8af5fbf8c4e2bcdaaa3e05e2f0705d179
